src/utils.ts

```diff
@@ -101,6 +109,12 @@ export function getSearchParam(key: string, location: RouteLocation): string | n
   return params.get(key);
 }
 
+// Checks an array for a pattern, if found returns element else returns undefined
+export function hasArrayElement(array: string[], pattern: string): string | undefined {
+  const reg = new RegExp(pattern);
+  return array.find(a => reg.test(a));
+}
```

Comment:
I find this util function a little dubious, based on the two places it's used. I think it's perfectly fine to use `Array::find()` instead, we don't really save much!

Reply:
Yeah it's a reminiscent of the need to return `null` I think we can remove it safely